src/p_local.h

```diff
@@ -437,6 +437,7 @@ enum EDmgFlags
 	DMG_FOILBUDDHA = 128,
 	DMG_NO_PROTECT = 256,
 	DMG_USEANGLE = 512,
+	DMG_NO_PAIN = 1 << 10,
```

Comment:
Styling should be consistent, I know we discussed preferring bit shifts but let's keep things consistent as we can. This value needs to be 1024.